packages/react-native-codegen/android/gradlePlugin-build/gradlePlugin/src/main/java/com/facebook/react/codegen/plugin/CodegenPlugin.java

```diff
@@ -62,12 +64,13 @@ public void apply(final Project project) {
 
               ImmutableList<String> execCommands =
                   new ImmutableList.Builder<String>()
-                      .add("yarn")
+                      .add(os.contains("windows") ? "yarn.cmd" : "yarn")
```

Comment:
Just confirming there's no more "canonical" check for Windows that can be done here? 😀

Reply:
I have looked on StackOverflow, and comparing string prefixes seems to be the way to go. There's a wrapper function from Apache, but I checked they're doing the same thing under the hood. (+would need to add a dependency for it.)